test/org/sagebionetworks/bridge/services/ScheduledActivityServiceMockTest.java

```diff
@@ -372,111 +380,154 @@ public void deleteActivitiesForSchedulePlanRejectsBadValue() {
+        
+        verify(activityDao).saveActivities(scheduledActivityListCaptor.capture());
+        List<ScheduledActivity> saves = scheduledActivityListCaptor.getValue();
+        assertActivityGuids(saves, "AAA");
     }
     
     @Test
     public void persistedAndScheduledIncludedInResultsV3() {
-        List<ScheduledActivity> scheduled = createNewActivities("CCC");
-        List<ScheduledActivity> db = createStartedActivities("CCC");
-
-        List<ScheduledActivity> saves = merge(scheduled, db, V3_MERGE);
-        assertActivities(saves);
+        SchedulePlan aaa = schedulePlan("AAA");
+        SchedulePlan bbb = schedulePlan("BBB");
+        SchedulePlan ccc = schedulePlan("CCC");
+        
+        when(schedulePlanService.getSchedulePlans(ClientInfo.UNKNOWN_CLIENT, TEST_STUDY)).thenReturn(Lists.newArrayList(aaa,bbb,ccc));
+        when(activityDao.getActivities(eq(TIME_ZONE), any())).thenReturn(createStartedActivities("CCC"+TIME_PORTION));
+        
+        List<ScheduledActivity> returnedActivities = service.getScheduledActivities(createScheduleContext(NOW).build());
+        assertNotNull(returnedActivities.get(0).getStartedOn());
+        assertActivityGuids(returnedActivities, "AAA", "BBB", "CCC");
         
-        scheduled = service.orderActivities(scheduled, V3_FILTER);
-        assertNotNull(scheduled.get(0).getStartedOn());
+        verify(activityDao).saveActivities(scheduledActivityListCaptor.capture());
+        List<ScheduledActivity> saves = scheduledActivityListCaptor.getValue();
+        assertActivityGuids(saves, "AAA", "BBB");
     }
 
     @Test
     public void expiredTasksExcludedFromCalculationsV3() {
-        // create activities in the past that are now expired.
-        List<ScheduledActivity> scheduled = createExpiredActivities("AAA","BBB");
-        List<ScheduledActivity> db = createExpiredActivities("AAA","CCC");
+        SchedulePlan aaa = schedulePlan("AAA");
+        SchedulePlan bbb = schedulePlan("BBB");
+        SchedulePlan ccc = schedulePlan("CCC");
+        
+        when(schedulePlanService.getSchedulePlans(ClientInfo.UNKNOWN_CLIENT, TEST_STUDY)).thenReturn(Lists.newArrayList(aaa,bbb,ccc));
+        when(activityDao.getActivities(eq(TIME_ZONE), any())).thenReturn(createExpiredActivities("AAA"+TIME_PORTION,"CCC"+TIME_PORTION));
         
-        List<ScheduledActivity> saves = merge(scheduled, db, V3_MERGE);
-        assertActivities(saves);
+        List<ScheduledActivity> returnedActivities = service.getScheduledActivities(createScheduleContext(NOW).build());
+        assertActivityGuids(returnedActivities, "BBB");
         
-        scheduled = service.orderActivities(scheduled, V3_FILTER);
-        assertTrue(scheduled.isEmpty());
+        verify(activityDao).saveActivities(scheduledActivityListCaptor.capture());
+        List<ScheduledActivity> saves = scheduledActivityListCaptor.getValue();
+        assertActivityGuids(saves, "BBB");
```

Comment:
Changed test to do what the original test did, by asking for activities in the past that are expired.